projects/samples/robotbenchmark/maze_runner/plugins/robot_windows/maze_runner/maze_runner.js

```diff
@@ -9,39 +10,32 @@ webots.window('maze_runner').receive = function(message, robot) {
     time = parseFloat(message.substr(5));
     timeString = parseSecondsIntoReadableTime(time);
     $('#time-display').html(timeString);
-
-  } else if (message == 'stop') {
-    if (typeof sendBenchmarkRecord === "undefined" || !sendBenchmarkRecord(robot, this, benchmarkName, -time, metricToString)) {
-      $('#time-display').css('color','red');
-    }
-
+  } else if (message === 'stop') {
+    if (typeof sendBenchmarkRecord === 'undefined' || !sendBenchmarkRecord(robot, this, benchmarkName, -time, metricToString))
+      $('#time-display').css('color', 'red');
   } else if (message.startsWith('record:OK:')) {
-    $('#time-display').css('font-weight','bold');
+    $('#time-display').css('font-weight', 'bold');
     showBenchmarkRecord(message, benchmarkName, metricToString);
-
   } else if (message.startsWith('record:Error:')) {
-    $('#time-display').css('color','red');
+    $('#time-display').css('color', 'red');
     showBenchmarkError(message, benchmarkName);
-
   } else
     console.log("Received unknown message for robot '" + robot + "': '" + message + "'");
 
   function metricToString(s) {
     return parseSecondsIntoReadableTime(-parseFloat(s));
   }
 
-  function parseSecondsIntoReadableTime(s) {
-    var hours = s / 3600;
-    var absoluteHours = Math.floor(hours);
-    var minutes = (hours - absoluteHours) * 60;
+  function parseSecondsIntoReadableTime(timeInSeconds) {
+    var minutes = timeInSeconds / 60;
     var absoluteMinutes = Math.floor(minutes);
-    var m = absoluteMinutes > 9 ? absoluteMinutes : '0' +  absoluteMinutes;
+    var m = absoluteMinutes > 9 ? absoluteMinutes : '0' + absoluteMinutes;
     var seconds = (minutes - absoluteMinutes) * 60;
     var absoluteSeconds = Math.floor(seconds);
     var s = absoluteSeconds > 9 ? absoluteSeconds : '0' + absoluteSeconds;
-    var cs = Math.round((seconds - absoluteSeconds) * 100);
+    var cs = Math.floor((seconds - absoluteSeconds) * 100);
```

Comment:
Just to mention this is not equivalent.

Reply:
Yes.
The implementation of the `parseSecondsIntoReadableTime()` function is now exactly the same in all the JS files.